pkg/tournament/service.go

```diff
@@ -463,7 +463,7 @@ func directorOrAdmin(ctx context.Context, ts *TournamentService) (*entity.User,
 	return user, nil
 }
 
-func authenticateDirector(ctx context.Context, ts *TournamentService, id string, authenticateExecutive bool) error {
+func authenticateDirector(ctx context.Context, ts *TournamentService, id string, authenticateExecutive bool, reqString string) error {
```

Comment:
let's pass in the req as `proto.Message` and then just do log.Info().Interface("req", req), i think this is a little cleaner?